packages/ui/src/council/types/Candidate.ts

```diff
@@ -0,0 +1,25 @@
+import BN from 'bn.js'
+
+import { asMember, Member } from '@/memberships/types'
+
+import { CandidateFieldsFragment } from '../queries'
+
+export interface Candidate {
+  id: string
+  stakingAccount: string
+  rewardAccount: string
+  stake: BN
+  note: string
+  title: string
```

Comment:
So the candidacy will have:

*   title
*   summary
*   short description (bullet points, so `string[]`)

The `note` will be something completely different on the GraphQL probably (like `metadata`).

We could either:

*   add the fields below to the `Candidate` type
*   create `CandidateNote` type for the `Candidate.note` field

What will happen later on GraphQL should be contained in casting. Our components should use this type.